package.json

```diff
@@ -55,7 +55,7 @@
     "yarn": "^1.16.0"
   },
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "start": "adb reverse tcp:8081 tcp:8081 && react-native start",
```

Comment:
Have you investigated the difference between running `node_modules/react-native/local-cli/cli.js` and `react-native` directly?

Reply:
I'm actually not getting what this local-cli is for, because it is the same code as the normal cli that goes into `node_modules/.bin/`. I think it makes no difference.

Reply:
Ah now that I think of it I think it's just a way to avoid accidentally referencing a globally installed `react-native` from `$PATH`, and I think that is generally a good idea to keep it that way (`node_modules/react-native/local-cli/cli.js`) to avoid hitting weird unexpected bugs later on!

Reply:
I see your point of  global `react-native` conflicts, but we do use `react-native` everywhere in our scripts except `start`.

Reply:
yarn scripts (and I think npm as well?) run the version installed by the package manager in the directory, not the globally installed one.

If the script is: 

```
"start": "react-native start"
```

`yarn start` should find the react-native bin installed in the directory and use that to run `start`, so using it should be fine I think.